### ISSUE TITLE
Tiro breaks on links with custom URI schemes

### ISSUE THREAD

**Issue by jnordberg** [AlexxNB/tinro#34]
I have an app that uses a link with a custom url scheme and when it is clicked a error is thrown in tinro's click handler:

```
SecurityError: Blocked attempt to use history.pushState() to change session history URL from http://localhost:8080 to esr:[redacted]. Protocols, domains, ports, usernames, and passwords must match.
```

**Comment by AlexxNB** [comment]
can you provide an example of the link? Do you use  `<a>` element or navigate with `router.goto()`?

**Comment by jnordberg** [comment]
Any `<a>` link with a custom protocol will trigger it. E.g.` <a href="mailto:hello@example.com">Test</a>`

**Comment by AlexxNB** [comment]
Thanks! Should be fixed in v0.4.6

**Comment by jnordberg** [comment]
Confirmed fixed! Thanks!
